src/cpu/aarch64/acl_gemm_convolution_utils.cpp

```diff
@@ -162,42 +156,106 @@ status_t init_conf(acl_conv_gemm_conf_t &acp, memory_desc_t &src_md,
     const auto acl_layout = is_nspc ? arm_compute::DataLayout::NHWC
                                     : arm_compute::DataLayout::NCHW;
 
+    // int8?
+    acp.is_int8 = utils::one_of(src_d.data_type(), s8, u8)
+            && wei_d.data_type() == s8;
+
+    auto acl_src_data_t
+            = acl_gemm_convolution_utils::get_acl_data_t(src_d.data_type());
+    auto acl_wei_data_t
+            = acl_gemm_convolution_utils::get_acl_data_t(wei_d.data_type());
+    auto acl_dst_data_t
+            = acl_gemm_convolution_utils::get_acl_data_t(dst_d.data_type());
+
+    auto acl_bia_data_t = arm_compute::DataType::F32;
+    if (acp.is_int8) acl_bia_data_t = arm_compute::DataType::S32;
```

Comment:
why not just:
``` cpp
auto acl_bia_data_t = acl_gemm_convolution_utils::get_acl_data_t(bia_d.data_type());
```

This should be less error prone.

Reply:
This doesn't work because, as far as I understand, `bia_d` can still have `undef` data type in which case `get_acl_data_t` returns DataType::UNKNOWN and the function validation in Compute Library fails.

What sort of errors is this prone to cause?